src/expound/alpha.cljc

```diff
@@ -381,6 +381,9 @@
 
 (defn ^:private problem-type [failure problem]
   (cond
+    (:expound.spec.problem/type problem)
```

Comment:
This should probably be 
```
(when-let [typ (:expound.spec.problem/type problem)]
   (get-method expound/problem-group-str typ))
```
to ensure that the problem type is supported. 

If not fallback to expounds normal behavior.

Reply:
otherwise an error will be thrown for an unsupported type

Reply:
Good call.